front/app/components/HookForm/RadioGroup/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import Error, { TFieldName } from 'components/UI/Error';
+import { useFormContext } from 'react-hook-form';
+import { CLError } from 'typings';
+
+import Radio from './Radio';
+import { Box } from '@citizenlab/cl2-component-library';
+
+interface Props {
+  name: string;
+  children?: React.ReactNode;
+}
+
+const RadioGroup = ({ name, children }: Props) => {
```

Comment:
New RHF wrapper to handle radio inputs. Validation is handled in RadioGroup while the form control is done in the `Radio` components that should be nested inside.